python/python/lance/dataset.py

```diff
@@ -235,6 +235,7 @@ def to_batches(
         offset: Optional[int] = None,
         nearest: Optional[dict] = None,
         batch_readahead: Optional[int] = None,
+        ordered_scan: bool = True,
```

Comment:
What do we think of this name? Is there another we'd prefer?

Reply:
Sounds reasonable. 👍 

Or should we name this as `batch_ordered`? Will people think of this flag as `ORDER BY` kind of sorting semantic

Reply:
sgtm.